php/Terminus/Collections/SshKeys.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Terminus\Collections;
 
+use Terminus\Exceptions\TerminusException;
```

Comment:
When I tried to run this command locally, it failed because this exception was not found. This is probably testable, but is this perhaps part of the "just assume all inputs are valid" phase we are in? @ronan